app/components/dns-records-table.tsx

```diff
@@ -132,6 +133,7 @@ export default function DnsRecordsTable(props: DnsRecordsTableProps) {
                 const isLoading =
                   transition.state === 'submitting' &&
                   Number(transition.submission.formData.get('id')) === dnsRecord.id;
+                const renewalAllowed = dayjs(dnsRecord.expiresAt).isBefore(dayjs().add(6, 'month'));
```

Comment:
Since all boolean variables there has the suffix `is`, maybe it would be better to add it to yours too to keep consistency in naming. So `isRenewalAllowed`

Reply:
Hmmm, to me it reads better without the `is`. Do you think it would be better to remove the `is` from the other variables too?

Reply:
`isRenewable` would be good.  I'd say that the `is*` style is already in place, let's match it.

Reply:
Oh, `isRenewable` is good. I will change it